Rakefile

```diff
@@ -34,11 +34,16 @@ end
 
 task :submodules => ['_sass/brand/.git']
 
-task :dev => [:dependencies, :submodules] do
+task :styles => [:dependencies] do
+  highlight_css = `bundle exec rougify style github`.strip()
+  File.write("_sass/_syntax-highlighting.scss", highlight_css)
+end
```

Comment:
> Dynamically generate the syntax highlighting styles, so it's clear where they came from.

I'm not sure this makes it any clearer where the styles come from? Having a comment in the file or something might be an alternative (and would avoid additional compile-time overhead). Even if we also provide the command which generates the file and validate in CI that the file matches that command (the [validate-generated-files](https://github.com/marketplace/actions/validate-generated-files) action can help with the latter)

Reply:
The question is, where did the current file come from? It's hard to tell. No need for that if it's generated.

Reply:
Somewhat, though a comment at the top of the file like:
```
// This file was generated by running ....
```
Feels like it'd be at least as good?

Reply:
Yes, for the sake of source attribution alone, that's functionally the same. But this way, there's no longer a generated file committed into the repository, since it can be easily generated automatically at build time.